source/run-commands.txt

```diff
@@ -117,13 +117,30 @@ using the :option:`--norc <mongosh --norc>` option.
 Terminate a Running Command
 ---------------------------
 
-To terminate a running command or query in ``mongosh``,
+To terminate a running command or query in :binary:`~bin.mongosh`,
 press ``Ctrl + C``.
 
-``Ctrl + C`` terminates the process in the shell, but does not
-terminate the process on the MongoDB server. This behavior
-differs from the :method:`db.killOp()` method, which terminates
-commands on the server.
+When you enter ``Ctrl + C``, :binary:`~bin.mongosh`: 
+
+- interrupts the active command,
+- tries to terminate the running process, and
```

Comment:
I'm not sure the user (or me, tbh) would be entirely sure on the difference between these two points. Maybe instead of "running process", we could say "ongoing server-side operation" here? ("terminate a process" sounds like killing the shell process itself, which is not what happens)

Reply:
Done